cli/cli.js

```diff
@@ -45,11 +45,22 @@ const version = packageJson.version;
 const JHIPSTER_NS = CLI_NAME;
 
 const argBlueprints = loadBlueprints();
-const configBlueprints = loadBlueprintsFromYoRc().map(bp => bp.name);
-const allBlueprints = [...new Set([...argBlueprints, ...configBlueprints])];
+// Create a 'blueprintName: blueprintVersion' object.
+const versionedBlueprints = argBlueprints.reduce((versionedBlueprints, blueprint) => {
+    versionedBlueprints[blueprint] = undefined;
+    return versionedBlueprints;
+}, {});
+
+const blueprintsWithVersion = loadBlueprintsFromYoRc();
+blueprintsWithVersion.forEach(blueprint => {
+    versionedBlueprints[blueprint.name] = versionedBlueprints[blueprint.name] || blueprint.version;
```

Comment:
isn't `versionedBlueprints[blueprint.name]` always `undefined`? because of l.50: `versionedBlueprints[blueprint] = undefined;`